website/docs/prysm-usage/checkpoint-sync.md

```diff
@@ -4,68 +4,200 @@ title: Sync from a checkpoint
+
+<Tabs groupId="os" defaultValue="others" values={[
+    {label: 'Windows', value: 'win'},
+    {label: 'Linux, MacOS, Arm64', value: 'others'}
+]}>
+  <TabItem value="win">
+
+```
+./prysm.bat beacon-chain --checkpoint-sync-url=http://localhost:3500 --genesis-beacon-api-url=http://localhost:3500
+```
+    
+  </TabItem>
+  <TabItem value="others">
 
 ```bash
-$ ./prysm.sh beacon-chain --checkpoint-sync-url=http://localhost:3500 --genesis-beacon-api-url=http://localhost:3500
+./prysm.sh beacon-chain --checkpoint-sync-url=http://localhost:3500 --genesis-beacon-api-url=http://localhost:3500
 ```
 
-## Downloading Checkpoint data from the Beacon Node API
+  </TabItem>
+</Tabs>
 
-`prysmctl` provides a tool for downloading the ssz-encoded BeaconState and SignedBeaconBlock to be used for Checkpoint Sync. This tool can be used to share files without publicly exposing an API endpoint, or by a block explorer or client team who wants to host the files statically as a trusted source.
+To confirm that checkpoint sync has succeeded, look for the following output:
+
+```
+level=info msg="requesting <your configured checkpoint sync endpoint>"
+```
+
+### Serve checkpoint sync requests
+
+The above instructions tell you how to **request** checkpoint state from another node. If you want to **serve** these requests, run a fully synced node with the following flags:
+
+ - `--enable-debug-rpc-endpoints`: The [Beacon Node API for retrieving a BeaconState](https://ethereum.github.io/beacon-APIs/#/Debug/getStateV2) is a debug endpoint - this flag tells Prysm to enable the endpoint so checkpoint sync requests can be served through your beacon node's RPC gateway provider endpoint.
+ - `--grpc-max-msg-size=65568081`: By default, Prysm caps the size of its RPC responses. This flag configures a cap that allows Prysm to serve checkpoint sync requests with checkpoint state files.
+
+Note that **this is entirely optional**. The beacon node *requesting* the checkpoint state from this node doesn't need these flags.
+
+
+## Option 2 (Advanced): Configure checkpoint sync via file export/import
+
+:::info
+
+This method requires `go` to be installed on the machine that hosts your synced beacon node.
+
+:::
+
+When you sync via **network request**, the `BeaconState`, `SignedBeaconBlock`, and genesis state files are delivered from one beacon node to another using a peer-to-peer connection. When you sync via **file export/import**, you manually export these files from one beacon node and import them into another. This can be useful if you don't want your beacon node to expose an RPC gateway provider endpoint. Block explorers and client teams can also host these exported files statically as a trusted checkpoint sync source.
+
+Issue the following command to export the `BeaconState` and `SignedBeaconBlock` files from a synced beacon node using `prysmctl`:
```

Comment:
TODO: caveat needed - available only as part of v3 rc0?

Reply:
Once v3 is released and tagged in github you should be able to run prysmctl. I think we should probably update the go run command to `go run github.com/prysmaticlabs/prysm/cmd/prysmctl@latest`.